opslevel/resource_opslevel_user.go

```diff
@@ -41,23 +41,15 @@ func resourceUser() *schema.Resource {
 				Optional:     true,
 				ValidateFunc: validation.StringInSlice(opslevel.AllUserRole, false),
 			},
-			"skip_welcome_email": {
```

Comment:
I removed this because there's no way to read the value in state, so this variable should not be managed in terraform. (There is no `SkipWelcomeEmail` field on the  `User` resource)